### PR TITLE
[Optimize] Optimize some code of config controller

### PR DESCRIPTION
**Pull request by onewe**
- Remove unnecessary annotation `Autowired`
- Replace with <>
- Remove unnecessary init statement
- Change injection method from field injection to constructor injection
- Remove unnecessary bool string
- Remove unnecessary judgment statement

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by Yasoahmad — DISMISSED** [review]
`
`

**Comment by CHANDRAMA2** [comment]
Chandrama mahato

**Comment by li-xiao-shuang** [comment]
出现了一些冲突 需要解决下

**Comment by onewe** [comment]
@li-xiao-shuang done

**Review by li-xiao-shuang — APPROVED** [review]

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ClientMetricsController.java [hunk 1]
```diff
@@ -61,11 +60,14 @@
 @RequestMapping(Constants.METRICS_CONTROLLER_PATH)
 public class ClientMetricsController {
     
-    @Autowired
-    private ServerMemberManager serverMemberManager;
+    private final ServerMemberManager serverMemberManager;
     
-    @Autowired
-    private ConnectionManager connectionManager;
```
Comment (li-xiao-shuang) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ClientMetricsController.java:68]: 为什么要替换注入方式？

**Review by li-xiao-shuang — APPROVED** [review]

**Pull request merged**